### PR TITLE
add validation

### PR DESCRIPTION
**Pull request by reddish-purple-ocelot**
added max_length validation

### CODE REVIEW

**Review by Jesterovskiy — APPROVED** [review]

File: lib/frontend_api/validators/object_validator.rb [hunk 1]
```diff
@@ -91,6 +91,14 @@ def numeric(attrs, klass = Integer)
       end
     end
 
+    # predefined validation method
+    # checks that length of attr value is not longer then given length
+    def max_length(attrs, count)
+      Array(attrs).each do |attr|
+        error(attr, "is is too long (maximum is   %{@object.send(count)}  characters)") if @object.send(attr).size > @object.send(count)
```
Comment (KsYay) [inline lib/frontend_api/validators/object_validator.rb]: `#{}` instead of `%{}`

Comment (KsYay) [reply lib/frontend_api/validators/object_validator.rb]: maybe ```if @object.send(attr).size > count```?

Comment (reddish-purple-ocelot) [reply lib/frontend_api/validators/object_validator.rb]: You are right. I will change it.

**Review by KsYay — APPROVED** [review]

File: spec/validators/model_validator_spec.rb [hunk 2]
```diff
@@ -57,6 +57,36 @@ def foo(args); end
       end
     end
 
+    describe 'max_length' do
+      let(:validator_class) { Class.new(described_class) { validates :max_length, :foo, length=5 } }
```
Comment (Jesterovskiy) [inline spec/validators/model_validator_spec.rb:61]: You can use just 5

**Review by Jesterovskiy — APPROVED** [review]
Can you add validator here too https://github.com/Aejis/aejis_frontend_api/blob/master/spec/support/users/validators/user_validator.rb ?

File: lib/frontend_api/validators/object_validator.rb [hunk 3]
```diff
@@ -91,6 +91,14 @@ def numeric(attrs, klass = Integer)
       end
     end
 
+    # predefined validation method
+    # checks that length of attr value is not longer then given length
+    def max_length(attrs, length)
+      Array(attrs).each do |attr|
+        error(attr, "is is too long (maximum is #{length} characters)") if @object.send(attr).size > length
```
Comment (shhavel) [inline lib/frontend_api/validators/object_validator.rb:98]: a typo: two times `is` (`is is`)

Comment (reddish-purple-ocelot) [reply lib/frontend_api/validators/object_validator.rb:98]: Yes, you are right I missed it, thanks

**Pull request merged**